homesite-quote-conversion/homesite.py

```diff
@@ -1,4 +1,4 @@
-import pandas as pd
+##import pandas as pd
```

Comment:
hmm i think we still need pandas, right? 🤔

Reply:
Yea sorry about that. It's from me trying to get the packages working and ensuring I have them. I'll fix it and recommit the file.